contracts/DecentralizedKV.sol

```diff
@@ -131,7 +131,7 @@ contract DecentralizedKV is OwnableUpgradeable {
 
         if (paddr.hash == 0) {
             // append (require payment from sender)
-            _prepareAppend();
+            _prepareAppend(1);
```

Comment:
I wonder if function `_putInternal()` can be omitted since we can instead use something like `_putBatchInternal(_keys[0], _dataHashes[0], 1)` ?